core/src/main/java/com/datastrato/gravitino/storage/relational/service/RoleMetaService.java

```diff
@@ -56,7 +74,7 @@ public void insertRole(RoleEntity roleEntity, boolean overwritten) {
           roleEntity.namespace() != null
               && !roleEntity.namespace().isEmpty()
               && roleEntity.namespace().levels().length == 3,
-          "The identifier should not be null and should have three level.");
+          "The namespace of roleEntity should not be null and should have three level.");
```

Comment:
The namespace of RoleEntity must have three levels.

Reply:
user and group are also checked in this way.

 I would like to create another pr to solve this together. add `checkUser`,`checkGroup`,`checkRole` in `NameIdentifier`

Reply:
WDYT

Reply:
OK.